src/components/Stories/__stories__/Stories.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
+
+import {Button} from '../../Button';
+import {Stories, StoriesProps} from '../Stories';
+import {Story as MyStory} from '../types';
+
+export default {
+    title: 'Components/Stories',
+    component: Stories,
+} as Meta;
+
+const stories: MyStory[] = [
+    {
+        title: 'New navigation',
+        description:
+            'At the top of the panel is the service navigation for each service. ' +
+            'Below are common navigation elements: a component for switching between accounts ' +
+            'and organizations, settings, help center, search, notifications, favorites.',
+        url: 'https://yandex.eu',
+        media: {
+            url: 'https://storage.yandexcloud.net/uikit-storybook-assets/story-picture-2.png',
+        },
+    },
+    {
+        title: 'New navigation (2)',
+        description: 'A little more about the new navigation',
+        media: {
+            url: 'https://storage.yandexcloud.net/uikit-storybook-assets/sample_960x400_ocean_with_audio.mp4',
+            type: 'video',
+        },
+    },
+    {
+        title: 'New navigation (3)',
+        description: 'Switch to the new navigation right now',
+        media: {
+            url: 'https://storage.yandexcloud.net/uikit-storybook-assets/story-picture-4.png',
+        },
+    },
+];
+
+const DefaultTemplate: Story<StoriesProps> = (props: StoriesProps) => {
+    const [visible, setVisible] = React.useState(props.open);
+
+    React.useEffect(() => {
+        setVisible(props.open);
+    }, [props.open]);
+
+    return (
+        <React.Fragment>
+            <div>
+                <Button
+                    onClick={() => {
```

Comment:
Why not trigger it via storybook arguments?

Reply:
I don't quite understand what it's about

Reply:
```jsx
export default {
  argTypes: {
   visible: {
     type: 'boolean'
   }
  }
};

const Template = ({visible, ...args}) => visible ? <Stories {...args}/> : null;
```

Reply:
For clarity, I made it with a button by analogy with other components. But it works with storybook controls too.
<img width="541" alt="image" src="https://user-images.githubusercontent.com/4076124/163582186-3f3f4a51-224f-4fbd-b856-1d6cb5c9a40f.png">